go/vt/mysqlctl/backup.go

```diff
@@ -822,7 +822,7 @@ func Restore(
 	// without password, we are passing --skip-networking to greatly reduce the set
 	// of those who can connect.
 	logger.Infof("Restore: starting mysqld for mysql_upgrade")
-	err = mysqld.Start(context.Background(), "--skip-grant-tables", "--skip-networking")
+	err = mysqld.Start(context.Background(), true /* waitAsRoot */, "--skip-grant-tables", "--skip-networking")
```

Comment:
Hum looking at this a bit more, if I change this one to false (which is possible), the other place can be an internal start() call (it is done in Init), and I don't need to expose the flag at all... Let me explore that.